cypress.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: ['plugin:cypress/recommended'],
+  rules: {
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: ['**/*.cy.{js,ts,tsx}'],
+      },
+    ],
+  },
+  overrides: [
+    {
+      files: ['**/*.spec.*', '**/*.cy.*'],
+      rules: {
+        'jest/expect-expect': 'off',
+        'jest/valid-expect-in-promise': 'off',
+        'jest/valid-expect': 'off',
+        '@babel/no-unused-expressions': 'off',
+      },
+    },
+  ],
+}
```

Comment:
This will make it so that the deDeps will only override on spec and cy files, and then make them both devDeps

```suggestion
module.exports = {
  extends: ['plugin:cypress/recommended'],
  overrides: [
    {
      files: ['**/*.spec.*', '**/*.cy.*'],
      rules: {
        'jest/expect-expect': 'off',
        'jest/valid-expect-in-promise': 'off',
        'jest/valid-expect': 'off',
        '@babel/no-unused-expressions': 'off',
         'import/no-extraneous-dependencies': [ 'error', { devDependencies: ['**/*.cy.*','**/*.spec.*'] } ],
      },
    },
  ],
}
```